src/annotations/components/controlBar/AnnotationsControlBar.tsx

```diff
@@ -2,21 +2,15 @@
 import React, {FC} from 'react'
 
 // Components
-import {
-  ComponentSize,
-  FlexBox,
-  FlexBoxChild,
-  InfluxColors,
-  JustifyContent,
-  TextBlock,
-} from '@influxdata/clockface'
+import {ComponentSize, InfluxColors, TextBlock} from '@influxdata/clockface'
 
 import {useSelector} from 'react-redux'
 import {isAnnotationsModeEnabled} from 'src/annotations/selectors'
+import Toolbar from 'src/shared/components/toolbar/Toolbar'
 
 export const AnnotationsControlBar: FC = () => {
   const infoText1 =
-    'Click on a graph to create a point annotation, click + shift + drag to create a range annotation.'
+    'Shift + click on a graph to create a point annotation, click + shift + drag to create a range annotation.'
```

Comment:
should the `click + shift + drag to create a range annotation.` text be flagged behind `rangeAnnotations`?

Reply:
it will be; in the next PR that I do.

Reply:
see: https://github.com/influxdata/ui/pull/1761